src/components/Combobox/components/ComboboxOption/ComboboxOption.jsx

```diff
@@ -127,7 +127,6 @@ const ComboboxOption = ({
         role="row"
         aria-level={belongToCategory ? 2 : 1}
         aria-selected={isActive}
```

Comment:
combobox manage focus only visually when focus is on the search component. therefore tabindex=-1 is not needed